test/bitset.jl

```diff
@@ -65,11 +65,11 @@ end
     @test !(-1 in BitSet(1:10))
 end
 
-# # issue #8570
-# This requires 2^29 bytes of storage, which is too much for a simple test
-# s = BitSet(typemax(Int32))
-# @test length(s) === 1
-# for b in s; b; end
+@testset "issue #8570" begin
+    @test 400 > @allocated s = BitSet(typemax(Int32))
```

Comment:
There's some scoping issues here

Reply:
Yes, thanks. I missed this locally due to iterative evaluation + persistent global variables.